integreat_cms/cms/fixtures/test_data.json

```diff
@@ -3390,6 +3390,52 @@
       "embedded_offers": []
     }
   },
+  {
+    "model": "cms.page",
+    "pk": 30,
+    "fields": {
+      "created_date": "2023-08-18T10:31:48.247Z",
+      "lft": 1,
+      "rgt": 2,
+      "tree_id": 11,
+      "depth": 1,
+      "parent": null,
+      "region": 1,
+      "explicitly_archived": true,
```

Comment:
```suggestion
      "explicitly_archived": false,
```

Technically, this bulk action should not be available for archived pages (I still works for them though ^^). But I think it would be better to test the bulk action only for pages that are not archived, as that is the intended use case

Reply:
page has been un-archived.